### PR TITLE
Add new Zadara-CSI driver to driver.md

### PR DESCRIPTION
**Pull request by Levp-zadara**
CSI Driver for Zadara VPSA Storage Array & VPSA All-Flash
https://github.com/zadarastorage/zadara-csi

```release-note
NONE
```

### CODE REVIEW

**Comment by chengyu-l** [comment]
How did you modify the label of PR to **_release-note-none_**？

**Comment by Levp-zadara** [comment]
@chengyu-l you can add this to PR:
~~~
```release-note
 NONE
```
~~~
https://github.com/kubernetes/community/blob/master/contributors/guide/release-notes.md#applying-a-release-note

File: book/src/drivers.md [hunk 1]
```diff
@@ -60,6 +60,7 @@ Name | CSI Driver Name | Compatible with CSI Version(s) | Description | Persiste
 [Vault](https://github.com/kubevault/csi-driver) | `secrets.csi.kubevault.com` | v1.0 | A Container Storage Interface (CSI) Driver for mounting HashiCorp Vault secrets as volumes. | Ephemeral | N/A | N/A | 
 [vSphere](https://github.com/kubernetes-sigs/vsphere-csi-driver) | `vsphere.csi.vmware.com` | v1.0 | A Container Storage Interface (CSI) Driver for VMware vSphere | Persistent | Read/Write Single Pod | Yes | Raw Block
 [YanRongYun](http://www.yanrongyun.com/) | ? | v1.0 | A Container Storage Interface (CSI) Driver for YanRong YRCloudFile Storage  | Persistent | Read/Write Multiple Pods | Yes | 
+[Zadara-CSI](https://github.com/zadarastorage/zadara-csi) | ? | v1.0, v1.1 | A Container Storage Interface (CSI) plugin for Zadara VPSA Storage Array & VPSA All-Flash  | Persistent | Read/Write Multiple Pods | Yes | Raw Block, Snapshot, Expansion, Cloning
```
Comment (saad-ali) [inline book/src/drivers.md]: Please replace `?` with the name of your CSI driver as referenced by StorageClass. e.g. `csi.zadara.com`

Comment (Levp-zadara) [reply book/src/drivers.md]: Updated. Thanks!

**Comment by Levp-zadara** [comment]
@msau42 @saad-ali 
Is there something missing from my side? 
It's my 1st PR, I'm not fully familiar with PR process.

**Pull request merged**